nginx/Dockerfile-nginx.template

```diff
@@ -0,0 +1,11 @@
+FROM nginx:1.11
```

Comment:
I really want to get away from mstealy's use of "dockerfile-templates" and shell scripts that modify them. Let's just have actual dockerfiles everywhere and none of the dynamic templating of dockerfiles. this is a pretty strong requirement unless it's a huge time investment. please refactor or let's discuss.

Reply:
Yes, in this PR, we are able to fix it to be Dockerfile-nginx, no need customizable